src/groth16/aggregate/prove.rs

```diff
@@ -224,6 +228,14 @@ fn gipa_tipp_mipp<E: Engine>(
             // u_r = c[:n'] * v[n':]
             let tuc_r = commit::single_g1::<E>(&rvk_right, rc_left)
         };
+        let tab_l = tab_l?;
+        let tab_r = tab_r?;
+        let zab_l = zab_l?;
+        let zab_r = zab_r?;
+        let zc_l = zc_l?;
+        let zc_r = zc_r?;
+        let tuc_l = tuc_l?;
+        let tuc_r = tuc_r?;
```

Comment:
It would really pay off in code clarity to improve the `par!` macro so this dance was not needed.